docker-compose.yml

```diff
@@ -70,3 +70,4 @@ services:
       - JWT_PUBLIC_KEY_PATH=/opt/app/.cert/public.pem
       - JWT_PRIVATE_KEY_PATH=/opt/app/.cert/key.pem
       - GROWTHBOOK_CLIENT_KEY='local'
+      - MOCK_USER_ID='404'
```

Comment:
I think we should probably add another user for this, the 404 is technically a deleted user and might eventually be unclear to people/cause issues.
Let me know what you think?

Reply:
Ah, in that case, yes, it is probably better to introduce a more defined user for this purpose.